File: carmel-linx-laravel/patch_print_student.py
import os

with open('app/Http/Controllers/MentoringController.php', 'r', encoding='utf-8') as f:
    content = f.read()

old = "$data = (array) ($responseData->data ?? $responseData);"
new = "$data = (array) ($responseData->data ?? $responseData);\n        $data['student'] = $data['profile'] ?? null;"

content = content.replace(old, new)

with open('app/Http/Controllers/MentoringController.php', 'w', encoding='utf-8') as f:
    f.write(content)

print("Patched MentoringController print mapping")

File: carmel-linx-laravel/patch_print_student.php
<?php

\ = file_get_contents('app/Http/Controllers/MentoringController.php');

\ = "\ = (array) (\->data ?? \);";
\ = "\ = (array) (\->data ?? \);\n        \['student'] = \['profile'] ?? null;";

\ = str_replace(\, \, \);
file_put_contents('app/Http/Controllers/MentoringController.php', \);
echo 'Patched MentoringController print mapping';
